vulnerabilities/api.py

```diff
@@ -168,8 +170,8 @@ def bulk_search(self, request):
                 purl_response["resolved_vulnerabilities"] = []
                 purl_response["purl"] = purl_string
             response.append(purl_response)
-
-        return Response(response)
+        res = {"result": response}
```

Comment:
This is still nested.

Reply:
> This is still nesting the results.

As mentioned here https://drf-spectacular.readthedocs.io/en/latest/drf_spectacular.html#drf_spectacular.utils.extend_schema
to listing object we can use `Serializer(many=True)`

`@extend_schema(request=PackageBulkRequestSerializer, responses=PackageSerializer(many=True))`

When I am trying to return a list of Packages then it automatically convert it into paginator view.

Output : 

![image](https://user-images.githubusercontent.com/57043342/113826944-5d407900-97a0-11eb-951f-22ea385850f4.png)

And I also tried to remove pagination from `setting.py` file and add a custom paginator in other API's but then it adds unwanted parameters in bulk_api as it fetches directly from the Package Model.

Preview :

![image](https://user-images.githubusercontent.com/57043342/114741145-1c300200-9d68-11eb-9de5-7e51e915af1a.png)

Still trying some other options..